lib/periodicChecks.js

```diff
@@ -0,0 +1,45 @@
+const assignUnassignedReviewers = async (pullRequest, context) => {
```

Comment:
Do we need the LICENCE comment here?
/cc @seanlip

Reply:
I didn't find this in other files so I'm not sure it's needed.

Reply:
I had a discussion on this with Sean and he suggested to add the LICENCE in all such files.

Reply:
This should be to all files in this repo right?
Shouldn't that be done in a different PR?

Reply:
Let's add it for this file and create an issue (and create a PR whenever possible) to fix this in other files.

Reply:
Done.
The issue can be found at https://github.com/oppia/oppiabot/issues/56